api/src/services/auth.service.ts

```diff
@@ -195,6 +213,8 @@ export class AuthService {
       data: {
         activeAccessToken: accessToken,
         activeRefreshToken: newRefreshToken,
+        agreedToTermsOfService:
```

Comment:
How is this different from just `agreedToTermsOfService`?

Reply:
We don't want to set it to null if it wasn't passed in, i.e. if they had already accepted we won't send this field